sdk/template/template/src/generated/generatedClient.ts

```diff
@@ -7,9 +7,9 @@
  */
 
 import * as coreClient from "@azure/core-client";
-import * as Parameters from "./models/parameters";
-import * as Mappers from "./models/mappers";
-import { GeneratedClientContext } from "./generatedClientContext";
+import * as Parameters from "./models/parameters.js";
```

Comment:
we still need a way to handle this for future code generation, maybe via a option in the codegen tool

Reply:
Logged https://github.com/Azure/autorest.typescript/issues/1955